### ISSUE TITLE
Retinanet mAP 0 using resnet50 backbone with just one class annotated

### ISSUE THREAD

**Issue by esperancaleonardo** [fizyr/keras-retinanet#1255]
using this to train, can anyone help?

`CUDA_VISIBLE_DEVICES=0 python ./bin/train.py --steps 567 --epochs 40 --freeze-backbone --batch-size 1 --compute-val-loss --weighted-average --image-max-side 2500 csv ./train_anno.csv ./train_class.csv --val-annotations ./test_anno.csv'`

**EDIT**  Own dataset annotated with labelimage and xml converted with own script
